recipes/qhube-dask/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "qhub-dask-meta" %}
+{% set version = "2021.03.04" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask =2.14.0
+    - distributed =2.14
+    - dask-gateway =0.6.1
+
+test:
+  imports:
+    - dask
```

Comment:
change this test to something like `echo 'works'`

Reply:
Thanks, I was looking for something like that :smile:

Reply:
yep

```
test:
  commands:
    - echo 'works'
```